### DISCUSSION TITLE
가리기후강퇴

### DISCUSSION THREAD

**Discussion by gogop45** [storycraft/node-kakao#796] [Q&A]
가리기후 강퇴 정상구동되지만, 
>>>가리고내보내기후 내보내기해제하기
내보내기해제 구현은 어떻게 하나요? 4.4.0버전사용중입니다.

**Comment by enter-nickname** [comment]
말그대로 메세지가리기 -> 강퇴 -> 해제 하시면됩니다

**Comment by dbswlstjr2** [comment]
모듈 찾아보세요.. 모듈에 다 있는데 그걸 노력을 안하시고 물으시면 뭐.. 우리가 어찌 답장해야 되죠?
